src/pfe/portal/modules/LoadRunner.js

```diff
@@ -433,7 +433,7 @@ module.exports = class LoadRunner {
     let hashObject = { gitHash: hashValue };
     const filePath = path.join(this.workingDir, 'runinfo.json');
     try {
-      fs.writeJson(filePath, hashObject, { spaces: '  ' });
+      await fs.writeJson(filePath, hashObject, { spaces: '  ' });
```

Comment:
I wonder why our test for this function in LoadRunner.test.js was passing even though we weren't `await`ing this. Maybe just luck, because the test does an `await fs.readJson` after calling this function